libs/common/src/types/portal/activity.ts

```diff
@@ -0,0 +1,29 @@
+import { ActivityAuthor, ValuesOf } from '..';
+import { PORTAL_ACTIVITY_LABEL, PORTAL_ACTIVITY_TYPE } from '../../constants';
+
+export type ActivityType = ValuesOf<typeof PORTAL_ACTIVITY_TYPE>;
+export type ActivityLabel = ValuesOf<typeof PORTAL_ACTIVITY_LABEL>;
+
+/**
+ * NOTE: type and activityType are confusing.
+ * - type/ActivityType e.g "DEAL_CANCELLED"
+ * - activityType/ActivityLabel e.g "Deal cancelled"
+ * Because we have two fields using a "type" naming convention:
+ * - The "type" field is clear.
+ * - The "activityType" field uses an "ActivityLabel" type to help clarify.
+ * Ideally, we would run some data migration to rename one or both of these fields.
+ */
+export type Activity = {
```

Comment:
Sorry! Appreciate I'm commenting this quite late on in the PR process, but only just clocked that this will be exported from `@ukef/dtfs-common` without any context that its for portal - could you update it to `PortalActivity`?

Reply:
Great shout, have updated. Note - now, `ActivityAuthor` is the only one that does not have a "Portal" prefix because it is also used in TFM.